src/tests/fuzz/fuzz_predefpcap_aware.c

```diff
@@ -128,7 +129,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
     while (r > 0) {
         if (PacketCopyData(p, pkt, header.caplen) == 0) {
             // DecodePcapFile
-            TmEcode ecode = tmm_modules[TMM_DECODEPCAPFILE].Func(&tv, p, dtv);
+            TmEcode ecode = DecodePcapFile(&tv, p, dtv);
```

Comment:
Not tested.

Reply:
CI still builds the targets right, you could then try with an old or new bug

Reply:
Not tested as in GitHub CI doesn't build these 2 targets.

Reply:
CIFuzz did not build them ?

Reply:
> CIFuzz did not build them ?

Does it have `HAS_FUZZPCAP`? If so, then yes.

Reply:
From https://github.com/OISF/suricata/actions/runs/8758399109/job/24039129891?pr=10919

> Fuzz targets: ['/github/workspace/build-out/fuzz_applayerparserparse_imap', '/github/workspace/build-out/fuzz_applayerparserparse_websocket', '/github/workspace/build-out/fuzz_applayerparserparse_quic', '/github/workspace/build-out/fuzz_mimedecparseline', '/github/workspace/build-out/fuzz_applayerparserparse_ftp-data', '/github/workspace/build-out/fuzz_applayerparserparse_http', '/github/workspace/build-out/fuzz_applayerparserparse_dns', '/github/workspace/build-out/fuzz_applayerparserparse_mqtt', '/github/workspace/build-out/fuzz_applayerparserparse_ftp', '/github/workspace/build-out/fuzz_applayerparserparse_snmp', '/github/workspace/build-out/fuzz_applayerparserparse_rdp', '/github/workspace/build-out/fuzz_applayerparserparse_dhcp', '/github/workspace/build-out/fuzz_applayerparserparse_tftp', '/github/workspace/build-out/fuzz_siginit', '/github/workspace/build-out/fuzz_applayerparserparse_ssh', '/github/workspace/build-out/fuzz_applayerparserparse_ike', '/github/workspace/build-out/fuzz_confyamlloadstring', '/github/workspace/build-out/fuzz_predefpcap_aware', '/github/workspace/build-out/fuzz_applayerparserparse_krb5', '/github/workspace/build-out/fuzz_sigpcap', '/github/workspace/build-out/fuzz_applayerparserparse_http2', '/github/workspace/build-out/fuzz_applayerparserparse_ntp', '/github/workspace/build-out/fuzz_applayerparserparse_sip', '/github/workspace/build-out/fuzz_applayerparserparse_telnet', '/github/workspace/build-out/fuzz_applayerparserparse_modbus', '/github/workspace/build-out/fuzz_applayerparserparse_dnp3', '/github/workspace/build-out/fuzz_applayerparserparse_pgsql', '/github/workspace/build-out/fuzz_applayerparserparse', '/github/workspace/build-out/fuzz_applayerparserparse_tls', '/github/workspace/build-out/fuzz_sigpcap_aware', '/github/workspace/build-out/fuzz_applayerparserparse_nfs', '/github/workspace/build-out/fuzz_decodepcapfile', '/github/workspace/build-out/fuzz_applayerparserparse_dcerpc', '/github/workspace/build-out/fuzz_applayerparserparse_enip', '/github/workspace/build-out/fuzz_applayerparserparse_rfb', '/github/workspace/build-out/fuzz_applayerparserparse_smb', '/github/workspace/build-out/fuzz_applayerparserparse_bittorrent-dht', '/github/workspace/build-out/fuzz_applayerprotodetectgetproto', '/github/workspace/build-out/fuzz_applayerparserparse_smtp']

fuzz_predefpcap_aware is here